server.js

```diff
@@ -145,6 +146,30 @@ server.register(require('vision'), function (err) {
     });
 });
 
+// Setup Crumb - double submit cookie for all post requests
+// below is an array of regular expressions containing the exclusions as
+// We cannot use the check with the fine uploader because the
+// XMLHttpRequest leaks the tokens.
+var csrf_check_skip = [
+    new RegExp('/file/choose'),
+    new RegExp('/public/.*')
+];
+
+// Register the crumb plugin which creates a csrf token
+// which is used to validate POST returns are not from domains
+// other than the primary domain
+server.register({
+    register: Crumb,
+    options: { skip:
```

Comment:
Is there an option to set the cookie expiration period?  Martin needs to document this.  Ideally it would be 24hrs along with the session cookie.